### PR TITLE
Added support to mongodb logging

### PR DESCRIPTION
**Pull request by mauriciofreshw**
Working well locally. The OpenShift part is still TBD, I see some mongo related stuff in the openshift folder but I'm a little bit confused about how it should be configured. Please, feel free to modify this branch @ashtonmeuser

### CODE REVIEW

File: .docker/mongo/databaseInit/initialSetup.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+echo 'Creating application user and db'
+
+mongo ${MONGO_INITDB_DATABASE} \
+        --host localhost \
+        --port 27017 \
+        -u ${MONGO_INITDB_ROOT_USERNAME} \
+        -p ${MONGO_INITDB_ROOT_PASSWORD} \
+        --authenticationDatabase admin \
+        --eval "db.createUser({user: '${MONGO_INITDB_ROOT_USERNAME}', pwd: '${MONGO_INITDB_ROOT_PASSWORD}', roles:[{role:'dbOwner', db: '${MONGO_INITDB_DATABASE}'}, {role:'dbOwner', db: 'TEST_DB'}]});"
```
Comment (fwkendall) [inline .docker/mongo/databaseInit/initialSetup.sh:11]: The hardcoded DB name here should be swapped out

File: server/logger.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+require('winston-mongodb');
 
 const nodeEnv = process.env.NODE_ENV || 'development';
-
 winston.add(new winston.transports.Console(nodeEnv === 'test' && { silent: true }));
 
+const dbServer = process.env.MONGO_HOST || 'mongodb';
+const dbPort = process.env.MONGO_PORT || '27017';
+const dbUser = process.env.MONGO_USER || 'admin';
+const dbPassword = process.env.MONGO_PASSWORD || 'development';
```
Comment (fwkendall) [inline server/logger.js:10]: I would remove the default values for user/pass - for local dev they should always come from a .env, IMO including them in code is just asking for trouble like accidental misconfiguration

File: .docker/mongo/databaseInit/initialSetup.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+echo 'Creating application user and db'
```
Comment (fwkendall) [inline .docker/mongo/databaseInit/initialSetup.sh:3]: Can probably update this

File: .docker/mongo/databaseInit/initialSetup.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// Example of javascript script for mongodb
+
+let res = [
+  //db.container.drop()
```
Comment (fwkendall) [inline .docker/mongo/databaseInit/initialSetup.js:4]: You mentioned this working locally... what would the purpose of this script be vs. initialSetup.sh ?

Comment (mauriciofreshw) [reply .docker/mongo/databaseInit/initialSetup.js:4]: Just a leftover from another project. Usually this is helpful to run some js code after spinning up a db.

File: .docker/mongo/databaseInit/initialSetup.sh (hunk 1 above)
Comment (mauriciofreshw) [reply .docker/mongo/databaseInit/initialSetup.sh:11]: good catch!

**Comment by ashtonmeuser** [comment]
Closing in favour of PR#84.

**Pull request closed without merging**